data/avro-schema/src/main/resources/avro/net/corda/data/membership/db/request/command/AddNotaryToGroupParameters.avsc

```diff
@@ -0,0 +1,13 @@
+{
+  "type": "record",
+  "name": "AddNotaryToGroupParameters",
+  "namespace": "net.corda.data.membership.db.request.command",
+  "doc": "Add a notary virtual node to the notary service in an existing set of group parameters.",
+  "fields": [
+    {
+      "name": "notary",
+      "doc": "Notary virtual node to be added to the group parameters",
+      "type": "net.corda.data.membership.PersistentMemberInfo"
```

Comment:
I would prefer that `MemberInfo` would be a type on its own, and then the `AddNotaryToGroupParameters` and `PersistentMemberInfo` would hold that type...

Reply:
If I understand the suggestion correctly, this would involve changing the `PersistentMemberInfo` schema which would affect several other classes. Would it be better to do that in a separate PR?

Reply:
Discussed offline with @yift-r3, resolving